### DISCUSSION TITLE
Templates

### DISCUSSION THREAD

**Discussion by madprops** [madprops/grasshopper#152] [General]
In `General` there's a new addlist to create `Templates`.

These allow to define every custom property, and be used to apply to the selected tabs.

For instance you select some tabs and run a template command and they will get all the defined custom properties, like color, title, etc.

![templates234](https://github.com/user-attachments/assets/009b9182-d523-4c4f-8d17-e441871ff083)

**Comment by N3C2L** [comment]
Wonderful idea, thank you. I could only test it briefly now because it's already late, but I've already found use cases for myself, such as applying my favorite tags or different note presets to tabs. The feature can save a lot of time and is a great thing. Thanks!  👌🏼

**Comment by madprops** [comment]
Added a button on domain rules addlist, at the bottom, to get filled by a template.
